src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageSectorSelection/tests/__snapshots__/ImageSectorSelection.test.js.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The component should render with image source 1`] = `
+<div
+  class="selection"
+/>
+`;
+
+exports[`The component should render with initial selection 1`] = `
+<div
+  class="selection"
+>
+  <div
+    class="selection"
+  >
+    <img
+      height="360"
+      src="//:0"
+      width="640"
+    />
+    <div
+      class="rectangle"
+      style="left: 100px; top: 66.66666666666667px; width: 500px; height: 266.6666666666667px;"
+    >
+      <div
+        class="resizeHandle"
+      />
+      <div
+        class="backdrop"
+        style="outline-width: 640px;"
+      />
+    </div>
+  </div>
+</div>
+`;
+
+exports[`The component should render with minWidth and minHeight 1`] = `
+<MockedImageSelection
```

Comment:
Info: This specific tests is not rendered to html on purpose, as the goal of the test is it to check whether or not the "minHeight" and "minWidth" properties get passed correctly to the RectangleSelection component.